entries/l/lego.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "LEGO": {
+    "domain": "lego.com",
+    "tfa": [
+      "email"
+    ],
+    "notes": "To turn on, Go to Account Overview > Personal & Address Details > LEGO Account Details > Security > Use two-factor authentication",
```

Comment:
```suggestion
```

None of our other sites have the instructions in the notes

Reply:
Good to know. I understand the notes section now and where it appears on the website. Will update any other PRs in the future accordingly.